components/progressbar/index.css

```diff
@@ -89,12 +95,37 @@ governing permissions and limitations under the License.
   --spectrum-progressbar-spacing-top-to-text: var(--spectrum-component-top-to-text-300);
 }
 
-@media (forced-colors: active) {
-  .spectrum-ProgressBar-track {
-    forced-color-adjust: none;
-    --highcontrast-progressbar-fill-color: ButtonText;
-    --highcontrast-progressbar-track-color: ButtonFace;
-    border: 1px solid ButtonText;
+.spectrum-Meter {
+  --spectrum-progressbar-size-default: var(--spectrum-meter-inline-size);
```

Comment:
Since Meter is technically a separate variant, it would probably be a good idea for these to have separate mods. So updating these to something like `--spectrum-progressbar-size-default: var(--mod-meter-size-default, var(--spectrum-meter-inline-size));` where the name would match the progress bar naming but allow for separate modifying

Reply:
this makes a lot of sense!